### PR TITLE
newarm (v1)

### PR DESCRIPTION
**Pull request by duane701**
Replace the old (intake) arm with the new intake arm.

### CODE REVIEW

File: src/Commands/SetShifter.cpp [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
+#include "SetShifter.h"
 
-ArmLevel::ArmLevel(Arms::ArmLevelPosition position)
-: mPosition(position),
-  count(0)
+SetShifter::SetShifter(Chassis::ShifterValue shifterValue)
+: mShifterValue(shifterValue)
 {
 	// Use Requires() here to declare subsystem dependencies
 	// eg. Requires(chassis);
```
Comment (Diego-Andrade) [inline src/Commands/SetShifter.cpp:7]: Requires is required!

File: src/Commands/ToggleArms.cpp [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
+Intake::ArmsValue value
+	= Robot::intake->IsArmUp() ? Intake::kDown: Intake::kUp;
+Robot::intake->SetArms(value);
 }
 
 // Make this return true when this Command no longer needs to run execute()
-bool DefaultShooter::IsFinished()
+bool ToggleArms::IsFinished()
 {
 	return false;
```
Comment (Diego-Andrade) [inline src/Commands/ToggleArms.cpp]: Going to keep toggling, should be true

File: src/Commands/SetShifter.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#ifndef SetShifter_H
+#define SetShifter_H
+
+#include "Commands/Subsystem.h"
```
Comment (Diego-Andrade) [inline src/Commands/SetShifter.h]: Should include command.h, not subsystem.h

File: src/Commands/SetArms.h [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#ifndef SetArms_H
+#define SetArms_H
+
+#include "Commands/Subsystem.h"
```
Comment (Diego-Andrade) [inline src/Commands/SetArms.h]: Should include command.h not subsystem.h

File: src/Commands/ToggleArms.h [hunk 5]
```diff
@@ -1,13 +1,13 @@
-#ifndef ToggleKickstand_H
-#define ToggleKickstand_H
+#ifndef ToggleArms_H
+#define ToggleArms_H
 
 #include "Commands/Subsystem.h"
```
Comment (Diego-Andrade) [inline src/Commands/ToggleArms.h]: Should include command.h not substem.h

File: src/Robot.h [hunk 6]
```diff
@@ -37,7 +36,6 @@ class Robot : public IterativeRobot {
   static std::shared_ptr<Shooter> shooter;
```
Comment (Diego-Andrade) [inline src/Robot.h:24]: Fix spacing

**Review by Diego-Andrade — CHANGES_REQUESTED** [review]
FTP

File: src/Commands/IntakeOn.cpp [hunk 7]
```diff
@@ -1,9 +1,8 @@
 #include "IntakeOn.h"
 
-IntakeOn::IntakeOn(double horizontalSpeed, double verticalSpeed)
+IntakeOn::IntakeOn(double Speed)
 {
-	mHorizontalSpeed = horizontalSpeed;
-	mVerticalSpeed = verticalSpeed;
+	mSpeed = Speed;
```
Comment (Diego-Andrade) [inline src/Commands/IntakeOn.cpp]: Use member initialization list

File: src/OI.cpp [hunk 8]
```diff
@@ -21,11 +9,8 @@
 #include "Commands/PrepShooter.h"
 #include "Commands/ToggleArms.h"
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
-
 OI::OI() {
     // Process operator interface input here.
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
     driver.reset(new Joystick(0));
```
Comment (Diego-Andrade) [inline src/OI.cpp:14]: Do we want a const for driver as well? To be consistant

File: src/OI.cpp [hunk 9]
```diff
@@ -68,14 +53,10 @@ OI::OI() {
...
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=FUNCTIONS
-
 std::shared_ptr<Joystick> OI::getDriver() {
    return driver;
 }
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=FUNCTIONS
-
 double OI::getDriverLeftAxis() const {
 	return this->driver->GetRawAxis(1);
```
Comment (Diego-Andrade) [inline src/OI.cpp]: `this->` is not necessary. Also magic number?

File: src/Robot.cpp [hunk 10]
```diff
@@ -32,10 +21,7 @@ void Robot::RobotInit() {
 	oi.reset(new OI());
```
Comment (Diego-Andrade) [inline src/Robot.cpp]: Spacing not fixed

**Review by Diego-Andrade — CHANGES_REQUESTED** [review]
Spacing is still off, and some suggestions on having magic numbers

**Comment by duane701** [comment]
Re: Spacing is still off, and some suggestions on having magic numbers

Magic numbers will be fixed in a future PR.

PTAL

**Review by Diego-Andrade — APPROVED** [review]

**Comment by Diego-Andrade** [comment]
LGTM

**Pull request merged**